cmd/access/node_builder/access_node_builder.go

```diff
@@ -165,7 +165,7 @@ func DefaultAccessNodeConfig() *AccessNodeConfig {
 		apiBurstlimits:               nil,
 		PublicNetworkConfig: PublicNetworkConfig{
 			BindAddress: cmd.NotSet,
-			Metrics:     metrics.NewNoopCollector(),
+			Metrics:     metrics.NewNetworkCollector(zerolog.Logger{}),
```

Comment:
can you move this initialization to where the public network is set so it can use the node's actual logger